src/with_logabsdet_jacobian.jl

```diff
@@ -0,0 +1,43 @@
+function ChangesOfVariables.with_logabsdet_jacobian(::typeof(log1pexp), x::Real)
+    y = log1pexp(x)
+    return y, x - y
+end
+
+function ChangesOfVariables.with_logabsdet_jacobian(::typeof(logexpm1), x::Real)
+    y = logexpm1(x)
+    return y, (x - y)
+end
+
+function ChangesOfVariables.with_logabsdet_jacobian(::typeof(log1mexp), x::Real)
+    y = log1mexp(x)
+    return y, (x - y)
+end
+
+function ChangesOfVariables.with_logabsdet_jacobian(::typeof(log2mexp), x::Real)
+    y = log2mexp(x)
+    return y, (x - y)
+end
+
+function ChangesOfVariables.with_logabsdet_jacobian(::typeof(logit), x::Real)
+    y = logit(x)
+    y, log(inv(x * (1 - x)))
```

Comment:
A bit more efficient:
```suggestion
    y, -log(x * (1 - x))
```